tests/common/snappi_tests/snappi_test_params.py

```diff
@@ -56,3 +60,5 @@ def __init__(self):
         self.poll_device_runtime = True
         self.ecn_params = None
         self.traffic_flow_config = TrafficFlowConfig()
+        self.reboot_type = None
+        self.localhost = None
```

Comment:
do we need this? Isn't this just a fixture so it can be carried as a parameter in the relevant functions?

Reply:
this is for `localhost`

Reply:
We need `localhost` to call `reboot`. If I add `localhost` as a parameter to `run_traffic`, then I have to update all calls to this function since its signature has changed. It's called in several other places.